lib/classes/Utils.js

```diff
@@ -197,6 +200,25 @@ class Utils {
                   count: 1,
                 });
               }
+
+              // For HTTP events, see what authorizer types are enabled
+              if (event.http && event.http.authorizer) {
```

Comment:
Hey @awitherow, thanks for the comments. I'm a little confused on what you mean -- could you clarify? Are you saying we should refactor where the `authorizer` object is so we don't have to parse that far down? Or that I should remove that initial `if (event.http && event.http.authorizer)` check? If the latter, I initially had it that way, but it really made the checks pretty verbose, so I tried pulling those two checks up to the top. I'm open to discussion though 😄